LICENSE.md

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+=====================
+
+Copyright (c) 2015
```

Comment:
Worth bumping the year. Also, full copyright holder e.g org name `Copyright (c) 2018 UK Home Office`